net/dns-over-https/Makefile

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2021 Martin Schneider <martschneider@google.com>
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dns-over-https
+PKG_SOURCE_DATE:=2021-09-29
+PKG_SOURCE_VERSION:=b338c7ae52bed0d1a5e85e30c76d023efb46580d
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/m13253/dns-over-https
+PKG_MIRROR_HASH:=ab15dbefac0c9bb0d5763749b532284568002d5b4796ff61b8ca3b51cae771e7
+
+PKG_MAINTAINER:=Martin Schneider <martschneider@google.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/dns-over-https/Default
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=IP Addresses and Names
+  URL:=https://github.com/m13253/dns-over-https
+endef
+
+define Package/dns-over-https
+  $(call Package/dns-over-https/Default)
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+  TITLE:=Client and server software to query DNS over HTTPS
+endef
+
+define Package/dns-over-https/description
+  Client and server software to query DNS over HTTPS, using Google DNS-over-HTTPS protocol and IETF DNS-over-HTTPS (RFC 8484).
+endef
+
+define Package/dns-over-https/install
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(CP) $(PKG_BUILD_DIR)/doh-client/doh-client $(1)/usr/sbin/
+	$(CP) $(PKG_BUILD_DIR)/doh-server/doh-server $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/etc/dns-over-https
+	$(CP) $(PKG_BUILD_DIR)/doh-client/doh-client.conf $(1)/etc/dns-over-https/doh-client.conf
+	$(CP) $(PKG_BUILD_DIR)/doh-server/doh-server.conf $(1)/etc/dns-over-https/doh-server.conf
+	$(INSTALL_DIR) $(1)/etc/init.d/
+	$(INSTALL_BIN) ./files/doh-client.init $(1)/etc/init.d/doh-client
+	$(INSTALL_BIN) ./files/doh-server.init $(1)/etc/init.d/doh-server
+endef
+
+$(eval $(call BuildPackage,dns-over-https))
```

Comment:
If you include the go pacakge makefiles, you can also add `$(eval $(call GoBinPackage,,dns-over-https))`.